iothub/device/src/TransportSettings/IotHubClientAmqpSettings.cs

```diff
@@ -57,9 +58,24 @@ public IotHubClientAmqpSettings(IotHubClientTransportProtocol transportProtocol
         /// </summary>
         /// <remarks>
         /// If incorrectly implemented, your device may fail to connect to IoT hub and/or be open to security vulnerabilities.
+        /// <para>
+        /// This feature is only applicable for HTTP, MQTT over TCP, MQTT over web socket, AMQP
+        /// over TCP. AMQP web socket communication does not support this feature. For users who want
+        /// this support over AMQP websocket, you must instead provide a <see cref="ClientWebSocket"/>
+        /// instance with the desired callback set.
+        /// </para>
         /// </remarks>
         public RemoteCertificateValidationCallback RemoteCertificateValidationCallback { get; set; }
 
+        /// <summary>
+        /// An instance of client web socket to be used when transport protocol is set to web socket.
+        /// </summary>
+        /// <remarks>
+        /// If not provided, an instance will be created from provided values for
+        /// <see cref="IotHubClientTransportSettings.Proxy"/> and <see cref="IotHubConnectionCredentials.Certificate"/>.
```

Comment:
The certificate information in IotHubConnectionCredentials isn't always set directly by the user, eg. in cases where they use an SDK implemented IAuthenticationMethod. I'd rather you just broadly specify that websocket options in transport settings will be used to initialize the ClientWebSocket, instead of calling out the specific options explicitly.